pkg/controller/node/nodecontroller.go

```diff
@@ -111,6 +113,11 @@ type NodeController struct {
 	nodeStore      cache.StoreToNodeLister
 
 	forcefullyDeletePod func(*api.Pod)
+	availableCIDRs      sets.Int
```

Comment:
Or let's forget about this comment. It's probably not worth the effort. Hash maps are generally expensive, AFAIK go implementation uses roughly x30 of the size of key-value, but this would put us in still reasonable amount of 4 * 30 * 3072 = 360kB.